tests/next_tests/unit_tests/embedded_tests/test_nd_array_field.py

```diff
@@ -468,10 +468,10 @@ def test_absolute_indexing_value_return():
     field = common._field(np.reshape(np.arange(100, dtype=np.int32), (10, 10)), domain=domain)
 
     named_index = ((IDim, 12), (JDim, 6))
-    value = field[named_index]
+    value = field[named_index].ndarray
 
-    assert isinstance(value, np.int32)
-    assert value == 21
+    assert isinstance(value, np.ndarray)
```

Comment:
Maybe we should clarify this (also @egparedes):
do we care what kind the buffer is? I think we only care that the field is still a field, but the buffer could be np.int32 conceptually.